postmaster/utils.py

```diff
@@ -255,44 +255,41 @@ def validate_wtforms_password(form, field):
         """
         username = form.username.data
         password = form.password.data
+        two_factor = form.two_factor.data if form.two_factor.data else None
 
         try:
             if form.auth_source.data == 'PostMaster User':
                 admin = models.Admins.query.filter_by(username=username, source='local').first()
-
                 if admin:
-
                     if admin.is_unlocked():
-
                         if bcrypt.check_password_hash(admin.password, password):
+                            if admin.otp_active:
```

Comment:
It's written the way it is so we can return messages based on if they even have otp_active or if their token was correct.